spring-cloud-kubernetes-fabric8-autoconfig/src/main/java/org/springframework/cloud/kubernetes/fabric8/Fabric8AutoConfiguration.java

```diff
@@ -123,22 +117,4 @@ public Fabric8PodUtils kubernetesPodUtils(KubernetesClient client) {
 		return new Fabric8PodUtils(client);
 	}
 
-	@Configuration(proxyBeanMethods = false)
```

Comment:
move to a separate config, that is auto-configured. This way, it is not part of the bootstrap phase and it is not picked-up.